demos/notifications/notifications.html

```diff
@@ -0,0 +1,69 @@
+<!doctype html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<title>CSS Chassis - Typography</title>
+	<meta name="description" content="Typography skeleton for styling">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<link rel="stylesheet" href="../../dist/css/chassis.css">
+	<link href="http://fonts.googleapis.com/css?family=Open+Sans:400,600,700,400italic,700italic" rel="stylesheet">
+  <link rel="stylesheet" href="normalize.css">
+  <link rel="stylesheet" href="cayman.css">
+
+</head>
+<body>
+  <section class="main-content">
+		<h1>Notifications</h1>
+		<hr>	
+  	<!-- Default Notification -->
+		<div class="notification notification-default">
+			<strong>Title!</strong> Alert body...
+		</div>			
+<pre><code>&lt;div class="notification notification-default"&gt;
```

Comment:
There are JS libraries we could use to do this for us, as I mentioned in the meeting, instead of having to type everything twice and hoping we typed it the same way each time.